inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -935,8 +1016,13 @@ There were `r fmt_count(linelist_cleaned, is.na(sex))` cases missing information
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+age_pyramid_under_two
+
+This chunk creates a separate age/sex pyramid for children under 2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 ```{r age_pyramid_under_two, warning=FALSE}
-# plot age pyramid under 2s
+
 filter(linelist_cleaned, age_years <= 2) %>%
   plot_age_pyramid(age_group = "age_group_mon",
                    split_by = "sex") +
```

Comment:
maybe create a thirt option with the stack_by - rather than having it commented out?

Reply:
or just remove

Reply:
removing